src/components/views/dialogs/InfoDialog.tsx

```diff
@@ -44,6 +44,7 @@ export default class InfoDialog extends React.Component<IProps> {
     };
 
     render() {
+        // FIXME: Using a regular import will break the app
```

Comment:
shouldn't this comment remain?

Reply:
_Remain_? The line is being added not being removed, what am I missing? :sweat_smile:

Reply:
oh dear, first PR review in the morning :')